homeworks/08-functional-reactive-programming/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
[Optional] If manifest is used then title could also be added here `Animal Control`.